test_main.c

```diff
@@ -17,13 +17,20 @@
 
 #include "tests.h"
 
+int runverbose = 1;
+
+const char* vers      = "\\0$VER: ast_tests 1.0.0";
+const char* min_stack = "$STACK:16384";
+
```

Comment:
Frustrated having to keep setting the stack in the CLI before running.  Stack cookie fixes that.  Added the stack flag to VAMOS too just so its clear what we need.   Added the version cookie too just since I was here.